awx/main/models/inventory.py

```diff
@@ -1797,3 +1815,651 @@ class Meta:
 
     def get_absolute_url(self, request=None):
         return reverse('api:inventory_script_detail', kwargs={'pk': self.pk}, request=request)
+
+
+# TODO: move to awx/main/models/inventory/injectors.py
```

Comment:
Other parts of our code base that could be refactored into files in the inventory folder:

 - https://github.com/ansible/awx/blob/1ece7645472df06f25559c78d008ed69f56e900b/awx/main/utils/mem_inventory.py
 - computed fields
 - load from db in import: https://github.com/ansible/awx/blob/1ece7645472df06f25559c78d008ed69f56e900b/awx/main/management/commands/inventory_import.py#L317
 - save to db in import: https://github.com/ansible/awx/blob/1ece7645472df06f25559c78d008ed69f56e900b/awx/main/management/commands/inventory_import.py#L862

Reply:
I'd focus on refactoring that as a _follow-up_ to this PR, personally so this change is more digestible.